packages/core/src/logger/loggerOptions.ts

```diff
@@ -0,0 +1,15 @@
+import { Level, LoggerInterface } from '../coreInterfaces';
+import { LoggingConfiguration } from './loggerConfiguration';
+
+export interface LoggingOptions {
```

Comment:
This interface doesn't belong in the Core library and should be removed. When you clean up this Core library PR, you will find that this type is not used inside the Core library. You will only need the `LoggingConfiguration` interface.